plugins/source/stripe/codegen/fixtures/main.go

```diff
@@ -25,7 +26,15 @@ func main() {
 		Resources: make(map[string]any),
 	}
 
-	extras := []any{
+	dataStructs := []any{
```

Comment:
These are required to generate the fixture file to use with stripe's mock server.